lib/google_search_data_viewer/keywords/keyword.ex

```diff
@@ -4,6 +4,8 @@ defmodule GoogleSearchDataViewer.Keywords.Keyword do
   alias GoogleSearchDataViewer.Keywords.Schemas.KeywordUpload
   alias GoogleSearchDataViewer.Repo
 
```

Comment:
Normally for the context module, we will name it as plural form `GoogleSearchDataViewer.Keywords.Keywords`, singular is for the Schema.

Reference: https://nimblehq.co/compass/development/code-conventions/elixir/phoenix/#business-domain

Reply:
We can do it on a separate PR. 👍

Reply:
> We can do it on a separate PR. 👍

Thanks Thanh. I will make a chore task for my next PR prior to parsing